source/src/drivers/cs_Watchdog.cpp

```diff
@@ -42,3 +45,23 @@ void Watchdog::start() {
 void Watchdog::kick() {
 	nrfx_wdt_channel_feed(_channelId);
 }
+
+void Watchdog::setOperatingStateToWriteOnTimeout(uint8_t* data, uint8_t dataSize) {
+	LOGv("Set microapp as dirty");
```

Comment:
Use a define so it can easily be enabled when debugging this module:
```suggestion
	LOGWatchdogDebug("Set microapp as dirty");
```